src/api/sync.rs

```diff
@@ -84,6 +85,10 @@ pub enum ApiError {
     /// We tried to download chunk too many times
     #[error("Too many retries: {0}")]
     TooManyRetries(Box<ApiError>),
+
+    /// serde_json error
+    #[error("Serde json error: {0}")]
+    SerdeJson(#[from] serde_json::Error),
```

Comment:
How were we getting away without it before, is it the map(Box?) ?

Reply:
`.into_json` returns an `io::Result<T>` while `serde_json::from_value` returns a `Result<T, serde_json::Error>`

Reply:
And `reqwest::Response::json` returns a `Result<T, reqwest::Error>`